pkg/build/pipelines/ruby/install.yaml

```diff
@@ -0,0 +1,39 @@
+name: Install a ruby gem
+
+needs:
+  packages:
+    - busybox
+    - ca-certificates-bundle
+    - ruby-3.2
+
+inputs:
+  gem:
+    description: |
+      Gem name
+    required: true
+
+  version:
+    description: |
+      Gem version to install. This can be a version tag (v1.0.0)
+    required: true
+
+pipeline:
+  - runs: |
+      TARGET_DIR_BIN="${{targets.destdir}}/usr/bin"
+      TARGET_DIR_INSTALL="${{targets.destdir}}$(ruby -e 'puts Gem.default_dir')/"
+
+      mkdir -p "${TARGET_DIR_BIN}"
+      mkdir -p "${TARGET_DIR_INSTALL}"
+      
+      gem install \
+      --install-dir ${TARGET_DIR_INSTALL}  \
+      --bindir ${TARGET_DIR_BIN} \
+      --no-document \
+      --ignore-dependencies \
+      --local \
+      --verbose \
+      -v ${{inputs.version}} \
+      ${{inputs.gem}}
```

Comment:
What do you think about changing this to 

```
${{inputs.gem}}-${{inputs.version}}.gem
```

That way the user doesn't need to know the structure of the gem filename? We could add an optional `inputs.gem-filename` in case the gemspec generates a non-standard filename for the gem

Reply:
Then the use would look like

```yaml
pipeline:
  uses: ruby/build
    gem: webrick
  uses: ruby/install
    gem: webrick
    version: ${{package.version}}
```

Reply:
nice, done